packages/main/src/List.ts

```diff
@@ -1146,6 +1145,7 @@ class List extends UI5Element {
 			e.stopImmediatePropagation();
```

Comment:
this could be removed

Reply:
I’ve opened a [pull request](https://github.com/SAP/ui5-webcomponents/pull/10274) addressing this suggestion